packages/web-components/src/components/va-link/va-link.tsx

```diff
@@ -0,0 +1,169 @@
+import { Component, Event, EventEmitter, Host, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'va-link',
+  styleUrl: 'va-link.css',
+  shadow: true,
+})
+export class VaLink {
+  /**
+   * If `true`, the anchor text will be bolded and include a right arrow icon.
+   */
+  @Prop() active?: boolean = false;
+
+  /**
+   * If `true`, a channel icon will be displayed before the anchor text.
+   */
+  @Prop() channel?: boolean = false;
+
+  /**
+   * If `true`, the component-library-analytics event is disabled.
+   */
+  @Prop() disableAnalytics?: boolean = false;
+
+  /**
+   * If `true`, the link will be treated as a download, and a download icon will be displayed before the anchor text.
+   */
+  @Prop() download?: boolean = false;
+
+  /**
+   * The href attribute of the anchor.
+   */
+  @Prop() href!: string;
+
+  /**
+   * The suggested filename. Only valid if download is `true`.
+   */
+  @Prop() filename?: string;
+
+  /**
+   * The type of the file. Only displayed if download is `true`.
+   */
+  @Prop() filetype?: 'PDF';
```

Comment:
If this only accepts a PDF type, should we set a default?

Reply:
Our [guidance](https://design.va.gov/components/link/#download-link) hints that it's not limited to PDFs, but I haven't found a download link with a different file type. I could make this the default if we want that.

Reply:
Maybe making it a default isn't the answer. I'm a little concerned that limiting the type to PDF will cause problems, since the guidance is not limited to PDF. It just means that if someone wants to use this component with a different file type we'll have to update it, right?

Reply:
I added a prop named `abbrTitle` to get around this. 
```jsx
<va-link 
  abbr-title="Microsoft Word Document"
  download
  text="Download VA form 10-10EZ"
  filename="10-10EZ.docx"
  filetype="DOCX"
  pages="2"
/>
```

Reply:
This could possibly lead to inconsistent titles used for the `abbr` title, but I think it's an acceptable solution since we're not limiting the file extensions that can be used.